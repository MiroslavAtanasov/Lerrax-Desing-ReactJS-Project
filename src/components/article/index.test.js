import React from 'react'
import renderer from 'react-test-renderer'
import Article from '.'
import TestingEnvironment from '../../test-utils/router'


describe('Article component', () => {
    it('should render Article component', () => {
        const tree = renderer.create(
            <TestingEnvironment value={{
                user: {
                    loggedIn: true,
                    id: '123'
                }
            }}>
                <Article />
            </TestingEnvironment>
        ).toJSON()
        expect(tree).toMatchSnapshot()
    })
})