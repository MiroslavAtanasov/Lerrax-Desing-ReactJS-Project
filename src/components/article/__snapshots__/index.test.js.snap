// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Article component should render Article component 1`] = `
<div
  className="container"
>
  <img
    alt="article"
    className="image"
    onClick={[Function]}
  />
  <div>
    <p
      className="name"
    />
    <p
      className="price"
    >
      BGN 
    </p>
  </div>
</div>
`;
